app/assets/javascripts/components/stories/History/HistoryChanges/index.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import HistoryChange from './HistoryChange';
+import * as History from '../../../../models/beta/history';
+
+const HistoryChanges = ({ changes }) =>
+  History.unserializeChanges(changes).map(change =>
```

Comment:
This should be done inside the method called by the action as well, it's not a component's responsibility.

Also, side note: it'd be better to use `deserialize` instead of `unserialize` :smile: